.npmrc

```diff
@@ -1 +1,2 @@
 engine-strict=true
+legacy-peer-deps=true
```

Comment:
Please ensure a line ending.

If we have this here, do we still need all the command line arguments?

Reply:
Just wanted to really make sure the tests were running with this setting, I'll remove it from the cli args